Source/ThirdParty/libwebrtc/Configurations/sdp_integration_fuzzer.xcconfig

```diff
@@ -36,8 +36,7 @@ ALTERNATE_HEADER_SEARCH_PATHS = $(ALTERNATE_HEADER_SEARCH_PATHS_$(SDK_VARIANT));
 ALTERNATE_HEADER_SEARCH_PATHS_iosmac = $(BUILT_PRODUCTS_DIR)$(WK_ALTERNATE_FRAMEWORKS_DIR)$(WK_LIBRARY_HEADERS_FOLDER_PATH);
 
 HEADER_SEARCH_PATHS = $(inherited) $(ALTERNATE_HEADER_SEARCH_PATHS) $(BUILT_PRODUCTS_DIR)$(WK_LIBRARY_HEADERS_FOLDER_PATH);
+LIBRARY_SEARCH_PATHS = $(SDK_DIR)$(WK_LIBRARY_INSTALL_PATH) $(inherited);
```

Comment:
Putting the SDK path first seems concerning for at-desk builds, because you're telling the linker to prefer libraries from the SDK over anything else.
```suggestion
LIBRARY_SEARCH_PATHS = $(inherited) $(SDK_DIR)$(WK_LIBRARY_INSTALL_PATH);
```

You probably copied this line from JavaScriptCore's config, but I think it's incorrect there too :\